service/src/main/java/io/camunda/service/TenantServices.java

```diff
@@ -126,5 +131,9 @@ private TenantEntity getSingle(final TenantQuery query) {
     return tenantEntity;
   }
 
-  public record TenantDTO(Long key, String tenantId, String name) {}
+  public record TenantDTO(Long key, String tenantId, String name) {
+    public static TenantDTO fromEntity(final TenantEntity entity) {
```

Comment:
❓ Is this used in another PR/change?

Reply:
Yes, it's used [here](https://github.com/camunda/camunda/pull/26378/files#diff-3fb57e515445f5156d274608549bff8168069ac01a468b37e482f184646d7144R78).